execution/gethexec/express_lane_tracker.go

```diff
@@ -62,16 +63,78 @@ func NewExpressLaneTracker(
 		auctionContractAddr:  auctionContractAddr,
 		earlySubmissionGrace: earlySubmissionGrace,
 		chainConfig:          chainConfig,
+		useLogs:              false, // default to using logs
```

Comment:
Since this is a temporary code, I'm not sure if it's worth adding it to a config